### DISCUSSION TITLE
Translate to Portuguese

### DISCUSSION THREAD

**Discussion by ifurnas** [papermerge/papermerge-core#174] [Q&A] [answered]
Hello, I would like to help translate Papermerge to Portuguese? It is possible?

**Comment by ciur** [comment]
Currently there is no support for localization

**Reply by ifurnas** [reply]
Thanks @ciur

Do you have plans and an estimated date to make localization possible?

**Reply by ciur** [reply, marked as answer]
yes, there will be localization integration.
The localization will be introduced in 2.3.x version which will be available approximately in February 2024.
